docs/deployment/ha/kubernetes_ha.md

```diff
@@ -2,7 +2,7 @@
 title: "Kubernetes HA Services"
 nav-title: Kubernetes HA Services
 nav-parent_id: ha
-nav-pos: 2
+nav-pos: 6
```

Comment:
nit: I am not sure about why we have the `nav-pos` 6 for `kubernetes_ha.md` and 5 for `zookeeper_ha.md`. Does `2` and `1` also make sense?

Reply:
It does not make a difference since the relative positioning wrt `zookeeper_ha.md` is important. I'll update it in order to avoid future confusion, though.